allure-java-migrations/src/main/java/io/qameta/allure/aspects/Allure1Annotations.java

```diff
@@ -0,0 +1,205 @@
+package io.qameta.allure.aspects;
+
+import io.qameta.allure.model.Label;
+import io.qameta.allure.model.Parameter;
+import io.qameta.allure.model.TestResult;
+import org.apache.commons.lang3.reflect.FieldUtils;
+import org.aspectj.lang.reflect.MethodSignature;
+import ru.yandex.qatools.allure.annotations.Description;
+import ru.yandex.qatools.allure.annotations.Features;
+import ru.yandex.qatools.allure.annotations.Issue;
+import ru.yandex.qatools.allure.annotations.Issues;
+import ru.yandex.qatools.allure.annotations.Severity;
+import ru.yandex.qatools.allure.annotations.Stories;
+import ru.yandex.qatools.allure.annotations.TestCaseId;
+import ru.yandex.qatools.allure.annotations.Title;
+import ru.yandex.qatools.allure.model.DescriptionType;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.AnnotatedElement;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * Allure labels utils.
+ */
+public final class Allure1Annotations {
+
+    private static final String SEVERITY_LABEL = "severity";
+
+    private static final String FEATURE_LABEL = "feature";
+
+    private static final String STORY_LABEL = "story";
+
+    private static final String ISSUE_LABEL = "issue";
+
+    private static final String SUITE_LABEL = "suite";
+
+    private final MethodSignature signature;
+
+    private final Object target;
+
+    private final Object[] args;
+
+
+    public Allure1Annotations(final Object target, final MethodSignature signature, final Object... args) {
```

Comment:
That feels like you need to split this utility class into two different classes - one that extracts data and the other that creates labels etc